### PR TITLE
docs - awscc_lambda_function

### PR DESCRIPTION
**Pull request by naveen-amzn**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_lambda_function/lambda_file_system.tf [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# resource "awscc_iam_role" "main" {
```
Comment (wellsiau-aws) [inline examples/resources/awscc_lambda_function/lambda_file_system.tf:1]: this file is not used? perhaps delete it until its ready for future merge

File: templates/resources/lambda_function.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+To create a AWS lambda function with basic details
+{{ tffile (printf "examples/resources/%s/lambda_basic.tf" .Name)}}
+
+### Lambda Layer example
+To create a AWS lambda function using lambda layers
+{{ tffile (printf "examples/resources/%s/lambda_layer.tf" .Name)}}
+
+### Ephemeral storage example
+To create a AWS lambda function using Ephemeral storage
+{{ tffile (printf "examples/resources/%s/lambda_ephemral_storage.tf" .Name)}}
```
Comment (wellsiau-aws) [inline templates/resources/lambda_function.md.tmpl:24]: typo on the file name, I suggest to rename both the file and the path here
`lambda_ephemral_storage.tf`

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]

File: templates/resources/lambda_function.md.tmpl (hunk 2 above)
Comment (naveen-amzn) [reply templates/resources/lambda_function.md.tmpl]: updated filename and filepath from `lambda_ephemral_storage.tf` to `lambda_ephemeral_storage.tf`.

**Review by wellsiau-aws — APPROVED** [review]
LGTM

**Comment by marcosentino** [comment]
@naveen-amzn Thank you for the contribution!

**Pull request merged**